cvat/apps/dashboard/static/dashboard/js/dashboard.js

```diff
@@ -77,22 +77,21 @@ class TaskView {
 
     _upload(uploadAnnotationButton, format) {
         const button = $(uploadAnnotationButton);
-        $('<input type="file" accept="text/xml">').on('change', async (onChangeEvent) => {
-            const file = onChangeEvent.target.files[0];
-            $(onChangeEvent.target).remove();
-            if (file) {
-                button.text('Uploading..');
-                button.prop('disabled', true);
-                try {
-                    await this._task.annotations.upload(file, format);
-                } catch (error) {
-                    showMessage(error.message);
-                } finally {
-                    button.prop('disabled', false);
-                    button.text('Upload Annotation');
+        $('<input type="file" accept="text/xml,application/zip,application/json">')
```

Comment:
Again the code need to fix in several places if add 'txt', 'yaml', ... formats. Previously we accepted only xml and it was OK.